cpp/src/arrow/chunked_array_test.cc

```diff
@@ -68,6 +69,13 @@ TEST_F(TestChunkedArray, Make) {
   ASSERT_RAISES(Invalid, ChunkedArray::Make({chunk0}, int16()));
 }
 
+TEST_F(TestChunkedArray, MakeEmpty) {
+  ASSERT_OK_AND_ASSIGN(std::shared_ptr<ChunkedArray> empty,
+                       ChunkedArray::MakeEmpty(int64()));
+  AssertTypeEqual(*int64(), *empty->type());
+  ASSERT_EQ(empty->length(), 0);
```

Comment:
Can you also add a validation test? Should probably be `ASSERT_OK(empty->ValidateFull())`.